packages/netsuite-adapter/src/client/sdf_client.ts

```diff
@@ -167,15 +167,16 @@ export default class SdfClient {
   }
 
   @SdfClient.logDecorator
-  static async validateCredentials(credentials: SdfCredentials): Promise<AccountId> {
+  static async validateCredentials(credentials: SdfCredentials): Promise<AccountInfo> {
     const netsuiteClient = new SdfClient({
       credentials,
       globalLimiter: new Bottleneck(),
       instanceLimiter: () => false,
     })
     const { projectPath, authId } = await netsuiteClient.initProject()
     await netsuiteClient.projectCleanup(projectPath, authId)
-    return Promise.resolve(netsuiteClient.credentials.accountId)
+    const accountId = await Promise.resolve(netsuiteClient.credentials.accountId)
+    return { accountId }
```

Comment:
ummm, is this equivalent to
```suggestion
    return { accountId: netsuiteClient.credentials.accountId }
```
?

why do we need to go through Promise.resolve?

Reply:
You're right. I didn't want to change the existing implementation but it is redundant. I'll change that :)